core/src/main/java/com/scalar/db/transaction/consensuscommit/Snapshot.java

```diff
@@ -204,14 +203,7 @@ public Optional<Map<Snapshot.Key, TransactionResult>> getResults(Scan scan) thro
     if (!scanSet.containsKey(scan)) {
       return Optional.empty();
     }
-
-    Map<Key, TransactionResult> results = new LinkedHashMap<>();
-    for (Entry<Snapshot.Key, TransactionResult> entry : scanSet.get(scan).entrySet()) {
-      mergeResult(entry.getKey(), Optional.of(entry.getValue()))
-          .ifPresent(result -> results.put(entry.getKey(), result));
-    }
-
-    return Optional.of(results);
```

Comment:
Ditto. We no longer need to merge the scan result with the delete set since we now disallow scan after deletes.